docs/docs/product/roadmap.mdx

```diff
@@ -4,6 +4,32 @@ sidebar_position: 3
 
 # Roadmap
 
+## Prioritized roadmap activities for Q1 of 2024
+
+1. Edlib 3  
+	* Deploy to master 
+	* Remove old services from repo (WWW, URL-api resources, common, LTI, version, auth)
```

Comment:
missing comma between 'URL-api' and 'resources'

Reply:
Fixed.